openapi/code/named.go

```diff
@@ -1,347 +0,0 @@
-package code
-
-import (
-	"fmt"
-	"regexp"
-	"sort"
-	"strings"
-	"unicode"
-)
-
-var reservedWords = []string{
-	"break", "default", "func", "interface", "select", "case", "defer", "go",
-	"map", "struct", "chan", "else", "goto", "switch", "const", "fallthrough",
-	"if", "range", "type", "continue", "for", "import", "return", "var",
-	"append", "bool", "byte", "iota", "len", "make", "new", "package",
-}
-
-// Named holds common methods for identifying and describing things
-type Named struct {
-	Name        string
-	Description string
-}
```

Comment:
I rely on this in a couple of tools. This change is too big. Please setup https://github.com/google/copybara to sync code as the stop-gap measure.